app/packs/src/apps/mydb/elements/details/reactions/variationsTab/ReactionVariations.js

```diff
@@ -359,10 +358,12 @@ export default function ReactionVariations({ reaction, onEditVariations }) {
           columnDefs={columnDefs}
           readOnlyEdit
           onCellEditRequest={updateRow}
+          onComponentStateChanged={() => gridRef.current.api.autoSizeAllColumns(false)}
```

Comment:
Note that this PR depends on #1714, since I'm using a method here that doesn't exist in the current version of `ag-grid` (29).